src/common/snippets/include/snippets/lowered/pass/init_live_ranges.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
Minor (BTW 2025 will be soon 😄 )
```suggestion
// Copyright (C) 2024 Intel Corporation
```

Reply:
Updated